arches/app/templates/views/components/datatypes/reference.htm

```diff
@@ -0,0 +1,41 @@
+{% load i18n %}
+{% load template_tags %}
+
+<!-- ko ifnot: $data.search -->
+<div class="node-config-item">
+    <div class="col-xs-12 control-label widget-input-label">
+        <span data-bind="text: $root.translations.controlledList"></span>
+    </div>
+    <!-- ko if: $data.controlledLists() -->
+        <select 
+            data-bind="
+                options: $data.controlledLists, 
+                optionsText: 'name', 
+                optionsValue: 'id', 
+                select2Query: {
```

Comment:
Would be good to have some validation against saving the node in the graph editor without a controlled list, which causes frontend TypeErrors later in the widgets.

Reply:
I added a method to the datatype to validate the config which gets called in graph.validate. This seems like something that we should be doing for other datatypes as well.